Dockerfile.deploy

```diff
@@ -8,59 +8,66 @@ ENV SWIG_FEATURES="-D__x86_64__"
 ENV OLYMPIA_UID=9500
 RUN useradd -u ${OLYMPIA_UID} -s /sbin/nologin olympia
 
-# Add nodesource repository and requirements
-ADD docker/nodesource.gpg.key /etc/pki/gpg/GPG-KEY-nodesource
+# Add support for https apt repos and gpg signed repos
 RUN apt-get update && apt-get install -y \
         apt-transport-https              \
         gnupg2                           \
     && rm -rf /var/lib/apt/lists/*
-RUN cat /etc/pki/gpg/GPG-KEY-nodesource | apt-key add -
+# Add keys and repos for node and mysql
+COPY docker/*.gpg.key /etc/pki/gpg/
+RUN APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn \
+    apt-key add /etc/pki/gpg/nodesource.gpg.key \
+    && APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn \
+    apt-key add /etc/pki/gpg/mysql.gpg.key
+COPY docker/*.list /etc/apt/sources.list.d/
 ADD docker/debian-buster-nodesource-repo /etc/apt/sources.list.d/nodesource.list
 ADD docker/debian-buster-backports-repo /etc/apt/sources.list.d/buster-backports.list
+ADD docker/debian-buster-mysql-repo /etc/apt/sources.list.d/buster-mysql.list
 
 # IMPORTANT: When editing this list below, make sure to also update
-# `Dockerfile`.
-RUN apt-get update && apt-get -t buster install -y \
+# `Dockerfile.deploy`.
+# Allow scripts to detect we're running in our own container and install
+# packages.
+RUN touch /addons-server-docker-container \
```

Comment:
Do we want that file in the image?

Reply:
Yes. The `Makefile` uses that to figure out if we're inside a container or not.

Reply:
mmm, but why are we adding this in `Dockerfile.deploy` though?  The deployment pipeline doesn't use `make` (or if it does it specifies the correct makefile explicitly)

Reply:
Just consistency, and ensuring if we do ask ops to run `make <something>` manually later on it works as expected.